DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -426,7 +426,7 @@ def execute( self ):
     for requestName in requestNames:
       request = self.getRequest( requestName )
       if not request["OK"]:
-        log.error( request["Message"] )
+        log.error( "Error getting request %s" % requestName, request["Message"] )
```

Comment:
It was a good occasion to move the variable part of the log message to the second argument completely